doc/siterc.tex

```diff
@@ -701,6 +701,28 @@ \subsection{[hosts]}
     \end{lstlisting}
 \end{myitemize}
 
+\subparagraph[{[[[[}SYSTEM{]]]]}execution time limit polling intervals]{[hosts] \textrightarrow [[HOST]] \textrightarrow [[[batch systems]]] \textrightarrow [[[[SYSTEM]]]] \textrightarrow execution time limit polling intervals}
+\label{ExecutionTimeLimitPollingIntervals}
+
+The intervals between polling after a task job (submitted to the relevant batch
+system on the relevant host) exceeds its execution time limit. The default
+setting of PT1M, PT2M, PT7M will roughly polls 1 minute, 3 minutes and 10
+minutes after the task job exceeds its execution time limit.
```

Comment:
(The setting is a list of intervals between polling, so the accumulated times in minutes are roughly 1, 1+2=3 and 1+2+7=10. I'll change the wording slightly to make this clearer.)